CHANGES.txt

```diff
@@ -77,7 +77,7 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Added marking of intermediate and and multi target nodes generated from SConf tests so that 
       is_conftest() is more accurate.
     - Added test for configure check failing to ensure it didn't break generating and running ninja.
-
+    - Added support for the PCH environment variable to support subst generators. 
```

Comment:
MongoDB is attempting to gain generator support for the PCH variable primarily to be able to easily disable PCH in any given configure check.